app/models/pipedrive_client.rb

```diff
@@ -46,6 +46,14 @@ def log(type, message, external_entity_name = nil, external_id = nil)
     raise message
    end
 
+  class << self
+
+    def authorization(email, password)
```

Comment:
I see, I like the idea even if it is not consistent with the "Ruby style guide". It might need to be changed in the future though, if the only class method needed is `authorization`